packages/camera/camera_android_camerax/lib/src/java_object.dart

```diff
@@ -0,0 +1,74 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/foundation.dart' show immutable;
+import 'package:flutter/services.dart';
+
+import 'camerax_library.pigeon.dart';
+import 'instance_manager.dart';
+
+/// Root of the Java class hierarchy.
+///
+/// See https://docs.oracle.com/javase/8/docs/api/java/lang/Object.html.
+@immutable
+class JavaObject {
+  /// Constructs a [JavaObject] without creating the associated Java object.
+  ///
+  /// This should only be used by subclasses created by this library or to
+  /// create copies.
+  JavaObject.detached({
+    BinaryMessenger? binaryMessenger,
+    InstanceManager? instanceManager,
+  }) : _api = JavaObjectHostApiImpl(
+          binaryMessenger: binaryMessenger,
+          instanceManager: instanceManager,
+        );
+
+  /// Global instance of [InstanceManager].
+  static final InstanceManager globalInstanceManager = InstanceManager(
+    onWeakReferenceRemoved: (_) {},
```

Comment:
This should be

```dart
onWeakReferenceRemoved: (int identifier) {
  JavaObjectHostApiImpl().dispose(identifier);
},
```

Reply:
Ah, you're right! Thanks for pointing that out!